inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -74,11 +76,39 @@ class StaticShapeNMS final : public StageNode {
         input5->serializeBuffer(serializer);
         outputData->serializeBuffer(serializer);
         outputDims->serializeBuffer(serializer);
+
+        if (tempBuffers().size())
+            tempBuffer(0)->serializeBuffer(serializer);
     }
 };
 
+bool isCMXEnough(int cmxSize, int numSlices, int spatDim, std::vector<int> bufferSizes) {
+    int curOffset = 0;
+    int curSlice = 0;
+
+    auto buffer_allocate = [&curOffset, &curSlice, &numSlices, cmxSize](int numBytes) {
```

Comment:
Nitpick: `const auto`.
You may also pass `cmxSize` by reference.

Reply:
fixed